src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -3908,5 +3927,45 @@ private static bool IsIntegerTypeSupportingPointerConversions(TypeSymbol type)
 
             return false;
         }
+
+        private bool HasImplicitSpanConversion(TypeSymbol source, TypeSymbol destination, ref CompoundUseSiteInfo<AssemblySymbol> useSiteInfo)
+        {
+            if (!Compilation.IsFeatureEnabled(MessageID.IDS_FeatureFirstClassSpan))
```

Comment:
>Compilation [](http://example.com/codeflow?start=17&length=11)

Is `Compilation` always non-null here?

Reply:
You're right, it doesn't seem fully guaranteed, I will check the compilation here.